content/en/docs/reference/scheduling/config.md

```diff
@@ -21,13 +21,13 @@ by implementing one or more of these extension points.
 You can specify scheduling profiles by running `kube-scheduler --config <filename>`,
 using the
 KubeSchedulerConfiguration ([v1beta1](/docs/reference/config-api/kube-scheduler-config.v1beta1/)
-or [v1beta2](/docs/reference/config-api/kube-scheduler-config.v1beta2/)) 
+or [v1beta2](/docs/reference/config-api/kube-scheduler-config.v1beta2/)) or [v1beta3](docs reference need to be generated)
```

Comment:
@tengqm - Can you let me know how to generate these file for 1.23. The same way you did it here- https://github.com/kubernetes/website/pull/29243

cc @damemi

Reply:
cc @sftim - Do you happen to know how do it?